hbase-common/src/main/java/org/apache/hadoop/hbase/io/encoding/DataBlockEncoder.java

```diff
@@ -50,9 +50,8 @@ void startBlockEncoding(HFileBlockEncodingContext encodingCtx, DataOutputStream
 
   /**
    * Encodes a KeyValue.
-   * @return unencoded kv size written
```

Comment:
In the Javadoc pls add a line which clearly says the need to call the EncodingState#postEncode() after the encoding step